buildSrc/src/main/kotlin/org/jetbrains/exposed/gradle/DBTestingPlugin.kt

```diff
@@ -68,23 +72,35 @@ class DBTestingPlugin : Plugin<Project> {
             val mariadb_v2 = register<DBTestWithDockerCompose>("mariadb_v2Test", Parameters("MARIADB", 3306)) {
                 testRuntimeOnly("org.mariadb.jdbc", "mariadb-java-client", Versions.mariaDB_v2)
             }
-
             val mariadb_v3 = register<DBTestWithDockerCompose>("mariadb_v3Test", Parameters("MARIADB", 3306)) {
                 testRuntimeOnly("org.mariadb.jdbc", "mariadb-java-client", Versions.mariaDB_v3)
             }
-
             val mariadb = register<Test>("mariadbTest") {
                 group = "verification"
                 delegatedTo(mariadb_v2, mariadb_v3)
             }
 
+            val testPartial = register<Test>("testPartial") {
+                group = "verification"
+                delegatedTo(
+                    h2_v2,
+                    sqlite,
+                    mysql80,
+                    postgres,
+                    mariadb_v3,
+                    sqlServer
+                )
+            }
+
```

Comment:
Why do we need this?

Reply:
We don't, you're right, I just thought it'd be nice to have a smaller group for a quick run, to mimic the previous `test`. Not even sure I would ever use it, so I'll remove it.